packages/reactivity/__tests__/effect.spec.ts

```diff
@@ -25,7 +25,18 @@ describe('reactivity/effect', () => {
     counter.num = 7
     expect(dummy).toBe(7)
   })
+  it('should not cause infinite loop when getter or setter operations occur in effect', () => {
```

Comment:
It already exists.
https://github.com/vuejs/vue-next/blob/master/packages/reactivity/__tests__/effect.spec.ts#L342-L388